bin/generator/icon.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
```

Comment:
Oh fun! I was *just* thinking about how useful a generator like this would be as I added in the `expand` icon recently 👏